Extending/Packages/Packages-on-Umbraco-Cloud/index.md

```diff
@@ -4,9 +4,42 @@ versionFrom: 8.0.0
 
 # Packages on Umbraco Cloud
 
-If you want to use or develop packages for Umbraco Cloud there are a few things to consider and be aware of. One such thing is that custom property editors may need a **ValueConnector** to transform their content between environments.
+If you want to use or develop packages for Umbraco Cloud there are a few things to consider and be aware of.
+The two most important things to know about are
+- [How you should store data on Cloud](#storing-data)
+- [Using custom property editors with Deploy](#valueconnectors)
 
-# ValueConnectors
+## Storing data
+
+When developing a package you will sometimes store data, this can be data in many forms - Umbraco schema / content, package settings, etc.
+
+When you develop a package for Umbraco Cloud there are a few things to be aware of when storing data, mainly whether you want that data to be specific to 1 environment or more.
+
+Let's take a look at the most common ways of storing data in packages - and what to watch out for on Cloud:
+
+### Package actions
+
+A [package action](../Package-Actions/custom-package-actions.md) are used to run some commands on package install and uninstall. One thing to watch out for with regards to package actions is that whatever code you run in this will only be executed on the environment you install the package on. So if you for example set up a database table in a package action that db table will not be generated in each environments db.
+If you need a db table in each environments db you will probably need to look at migrations below instead.
+
+### Migrations
+
+A [database migration](../../Database/index.md) is a set of database commands you run as part of a migration plan. That migration plan has an id that is stored in the database (in the KeyValue table). This means that when you add new migrations Umbraco will only execute the ones that came after the one with the stored id.
```

Comment:
Migrations are not limited to database operations. You can run just run code as well : https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Core/Migrations/Upgrade/V_8_0_1/ChangeNuCacheJsonFormat.cs

Especially because you mention doing things that generate a Umbraco Schema. This is normally not something that happens when you purely do database migrations, but when use the services (dictionary service, datatype service, ...)

I get the confusion. Because the migrations documentation is located under database.

Reply:
Ah yes, good point! I knew that we use migrations on Our to generate doc types, templates, data types, etc but didn't make the connection when writing this 😕

Thanks for pointing it out I will fix it up later today!